src/API/Site/Controllers/MerchantCenter/PriceBenchmarksController.php

```diff
@@ -0,0 +1,142 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\BaseController;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query\MerchantPriceSuggestionsQuery;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\MerchantPriceBenchmarks;
+use Automattic\WooCommerce\GoogleListingsAndAds\Internal\ContainerAwareTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\Internal\Interfaces\ContainerAwareInterface;
+use Exception;
+use WP_REST_Request as Request;
+use WP_REST_Response as Response;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class PriceBenchmarksController
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter
+ */
+class PriceBenchmarksController extends BaseController implements ContainerAwareInterface {
+
+	use ContainerAwareTrait;
+
+	/**
+	 * Register rest routes with WordPress.
+	 */
+	public function register_routes(): void {
+		$this->register_route(
+			'mc/price-benchmarks',
+			[
+				[
+					'methods'  => TransportMethods::READABLE,
+					'callback' => $this->get_price_benchmarks_callback(),
+				],
+				'schema' => $this->get_api_response_schema_callback(),
+			]
+		);
+	}
+
+	/**
+	 * Maps query arguments from the REST request.
+	 *
+	 * @param Request $request REST Request.
+	 * @return array
+	 */
+	protected function prepare_query_arguments( Request $request ): array {
+		$args = wp_parse_args(
+			array_intersect_key(
+				$request->get_query_params(),
+				$this->get_collection_params()
+			),
+			$request->get_default_params()
+		);
+
+		return $args;
+	}
+
+	/**
+	 * Get the callback function for the price benchmarks request.
+	 *
+	 * @return callable
+	 */
+	protected function get_price_benchmarks_callback(): callable {
+		return function ( Request $request ) {
+			try {
+				/** @var MerchantPriceBenchmarks $merchant */
+				$merchant = $this->container->get( MerchantPriceBenchmarks::class );
+
+				$benchmark_data = $merchant->get_benchmark_data( 'products', $this->prepare_query_arguments( $request ) );
```

Comment:
This method doesn't currently accept args. I assume it's just not implemented yet.

Reply:
Addressed in https://github.com/woocommerce/google-listings-and-ads/pull/2838/commits/62c4d22ddf74481f575b2f9a28268dc9686537f4